cocotb/drivers/xgmii.py

```diff
@@ -108,7 +108,7 @@ def value(self):
         NB clears the value.
         """
         self._value.integer = self._integer
-        self._integer = long(0)
+        self._integer = 0
```

Comment:
This file was unusable on python 3. I can't see any reason why these would need to be `long` on python 2.